store/searchlayer/user_layer.go

```diff
@@ -33,7 +33,7 @@ func (s *SearchUserStore) deleteUserIndex(user *model.User) {
 	}
 }
 
-func (s *SearchUserStore) Search(teamId, term string, options *model.UserSearchOptions) ([]*model.User, *model.AppError) {
+func (s *SearchUserStore) Search(teamId string, term string, options *model.UserSearchOptions) ([]*model.User, error) {
```

Comment:
The duplicate `string` can be removed I think.

Reply:
Removed